### DISCUSSION TITLE
假的GPT4,大家不要上当受骗

### DISCUSSION THREAD

**Discussion by varsog** [timqian/openprompt.co#69] [Show and tell]
问题一：π和3.2哪个大？
GPT4：会说3.2大
它：π大。π的值约为3.14，大于3.2。

问题二：你是GPT4吗？
它：不，我是OpenAI的GPT-3。

此外，我尝试问第二类斯特林数的变种问题，它均回答错误（GPT4能正确回答）
如：8个不同的球放进3个相同的盒子里，允许盒子空着，有几种方法？

对于各类数学问题、逻辑问题的回答，均能看出不是GPT4。

**Comment by timqian** [comment]
<img width="898" alt="Screenshot 2023-09-03 at 12 33 40" src="https://github.com/timqian/openprompt.co/assets/5512552/84192f35-0893-465e-8e07-556e9f87858e">

使用 GPT-4 模型，需要在这里选择（注：付费用户才能使用）

**Reply by varsog** [reply]
![20230903124600](https://github.com/timqian/openprompt.co/assets/6064523/a69721c9-4c2f-4c8c-a6d8-389f6c456079)
我是付费用户，并且选择了GPT-4模型。
GPT4会自纠错，让它自检最终会肯定在197这个答案上，GPT3.5则不会。

**Comment by timqian** [comment]
如果在model 里面选择了 GPT-4，那么背后使用的就是 GPT-4 模型
